ext/bz2/bz2.c

```diff
@@ -339,8 +339,8 @@ static PHP_FUNCTION(bzread)
 	php_stream_from_zval(stream, bz);
 
 	if ((len + 1) < 1) {
-		php_error_docref(NULL, E_WARNING, "length may not be negative");
-		RETURN_FALSE;
+		zend_value_error("length cannot be negative");
+		return;
```

Comment:
If you have to push another commit, please change these `return`s to `RETURN_THROWS()` as well. If there is no need for pushing, I can do it after your changes are merged.

Reply:
I'll do it after fixing my Zlib PR :)